### PR TITLE
Fix isAdmin() breaking if cmd.exe has no output

### LINKED ISSUE #1255: isAdmin() causing client to crash at startup for Windows with broken cmd.exe
I have a case of a forum user that has a stack trace like this:

```
Exception in Application init method
java.lang.RuntimeException: Exception in Application init method
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(Unknown Source)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.util.NoSuchElementException: No line found
	at java.base/java.util.Scanner.nextLine(Unknown Source)
	at com.faforever.client.util.WindowsUtil.isAdmin(WindowsUtil.java:24)
	at com.faforever.client.FafClientApplication.init(FafClientApplication.java:74)
	... 3 more
```

The forum thread is here: http://forums.faforever.com/viewtopic.php?f=3&t=17576
After some brief discussion it seems user's cmd.exe is borked - it opens, then closes for a split second. I assume that also causes the client to fail to run the few script lines that the client needs. We need to catch this at least.

Do we want to depend on cmd.exe to figure out if the current user is an administrator? Maybe there's a better way, I don't know how much cmd.exe can be relied on.

### PR DESCRIPTION
**Pull request by Wesmania**
Fixes #1255.

Signed-off-by: Igor Kotrasinski <i.kotrasinsk@gmail.com>

### CODE REVIEW

**Comment by 1-alex98** [comment]
What about a catch Exception in genral to avoid any unexpected behaviours ?

**Comment by 1-alex98** [comment]
Have a look at Codacy and what it complain about

File: src/main/java/com/faforever/client/util/WindowsUtil.java [hunk 1]
```diff
@@ -21,6 +21,8 @@ public static boolean isAdmin() {
 
       boolean printedErrorlevel = false;
       while (true) {
+        if (!scanner.hasNextLine())     /* Unexpected output, bail out */
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/util/WindowsUtil.java]: use bracets

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

File: src/main/java/com/faforever/client/util/WindowsUtil.java (hunk 1 above)
Comment (1-alex98) [reply src/main/java/com/faforever/client/util/WindowsUtil.java]: and remove comment

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

File: src/main/java/com/faforever/client/util/WindowsUtil.java (hunk 1 above)
Comment (Wesmania) [reply src/main/java/com/faforever/client/util/WindowsUtil.java]: Bureaucracy :/ (Maybe if 2-space indents weren't the convention we wouldn't need that rule)

**Review by 1-alex98 — APPROVED** [review]

**Review by Brutus5000 — APPROVED** [review]

**Comment by norraxx** [comment]
why not
```
while(scanner.hasNextLine()) {
```
?

**Comment by 1-alex98** [comment]
Because we are only interested in the first one...

**Pull request merged**